onyo/commands/tests/test_new.py

```diff
@@ -86,6 +86,98 @@ def test_new_top_level(repo: OnyoRepo) -> None:
     fsck(repo)
 
 
+@pytest.mark.repo_dirs(*directories)
+def test_new_from_root_with_absolute_paths(repo: OnyoRepo) -> None:
```

Comment:
To be honest, I don't think we need this test. The very next test does essentially the same thing. Why would CWD being the root of the worktree be a special case when acting upon an absolute path? The command is supposed to act upon the given absolute path only - CWD should not somehow end up influencing the interpretation of that path. These things are supposed to be covered by `test_new_sub_dir_absolute_path`.

Reply:
Hm, I feel like you assume the way the paths are handled currently, and that it is done correctly everywhere (which I assume technically to be right).

But this test would IMO catch something like us at some place building paths like `Path(cwd, root_as_rel)`, that would let onyo look like it works always, but still would error when root would be an absolute path that gets entered wrong in some moment. So to test the correct passing around and handling/formatting of the path when absolute (instead of relative like normally) is here my intention.

Reply:
In any case: What makes `Path(cwd, root_as_rel)` special? Why does this deserve a dedicated test, that does the same thing as the next test for other directories in the worktree?